scripts/cocoapods/flipper.rb

```diff
@@ -92,3 +90,23 @@ def flipper_post_install(installer)
         end
     end
 end
+
+class FlipperConfiguration
+    attr_reader :flipper_enabled
+    attr_reader :configurations
+    attr_reader :versions
+
+    def initialize(flipper_enabled, configurations, versions)
+        @flipper_enabled = flipper_enabled
+        @configurations = configurations
+        @versions = versions
+    end
+
+    def self.enabled(configurations = ["Debug"], versions = {})
+        return FlipperConfiguration.new(true, configurations, versions)
+    end
+
+    def self.disabled
+        return FlipperConfiguration.new(false, [], {})
+    end
```

Comment:
Nit: In Ruby, the last statement is always returned so we can drop `return` here to be more idiomatic:
```suggestion
    def self.enabled(configurations = ["Debug"], versions = {})
        FlipperConfiguration.new(true, configurations, versions)
    end

    def self.disabled
        FlipperConfiguration.new(false, [], {})
    end
```